### PR TITLE
Update listener function naming convention

### PR DESCRIPTION
**Pull request by GoogleGenius**

### CODE REVIEW

**Comment by GoogleGenius** [comment]
I really want to tackle some refactoring with #51 later too, I don't really like how the if statements are nested in some areas.

File: src/index.tsx [hunk 1]
```diff
@@ -82,11 +82,11 @@ const initializeCourseViewer = (): void => {
...
   window.onscroll = () => {
-    displayDesktopScrollToTop();
-    handleMobileNavOnScroll();
+    handleDesktopScrollToTopDisplay();
+    handleMobileNavigationOnScroll();
   };
 
-  const displayDesktopScrollToTop = (): void => {
+  const handleDesktopScrollToTopDisplay = (): void => {
     if (!topButton)
```
Comment (Ascent817) [inline src/index.tsx:90]: Please add the curly braces here. This is javascript not python lol

Comment (GoogleGenius) [reply src/index.tsx:90]: I tend to disagree, it was supposed to be a one-liner except prettier decided to split it

Comment (Ascent817) [reply src/index.tsx:90]: But since it's not a one liner any more, just add the braces back.

Comment (GoogleGenius) [reply src/index.tsx:90]: FINE prettier does this for a bunch of other things too

Comment (Ascent817) [reply src/index.tsx:90]: Umm...The braces still aren't there lol

**Review by Ascent817 — APPROVED** [review]

**Pull request merged**